glean-core/src/upload/directory.rs

```diff
@@ -327,45 +346,6 @@ mod test {
         assert!(!wrong_contents_file_path.exists());
     }
 
-    #[test]
-    fn non_json_ping_body_files_are_deleted_and_ignored() {
```

Comment:
This is unrelated, after talking to @badboy we decided to stop checking if the ping body is valid JSON. It is very unlikely that it will be, since we are the ones writing to that file. Also it the ping will be rejected by the pipeline in the unlikely case it is not valid JSON.